### DISCUSSION TITLE
WYSIWYG Editor

### DISCUSSION THREAD

**Discussion by damionx7** [damionx7/Zettel-Notes-Documentation#113] [Feature Requests]
[Sam]: Add an option to edit in a WYSIWYG-style editor. On mobile, I find it far easier to edit markdown notes in something like UpNote's editor as opposed to a raw text editor.

**Comment by USCTrojan011** [comment]
Trying various different Android note apps, I think JotterPad has an excellent WYSIWYG editor that would be good to model after, except that it renders the `#` characters in headings.

**Comment by nathanl** [comment]
I don't mind this as long as using raw markdown is still an option. I prefer markdown to editors like Google Docs because of the precise control it gives. (Eg, ever fight Google Docs over multiple levels of nesting in a bulleted list?)
